astropy/io/misc/yaml.py

```diff
@@ -17,7 +17,7 @@
 - `astropy.coordinates.Latitude`
 - `astropy.coordinates.Longitude`
 - `astropy.coordinates.EarthLocation`
-- `astropy.table.SerializedColumn`
+- `astropy.table.SerializedColumn`.
```

Comment:
I would do a line break at l6 instead; the period here is wrong.

Reply:
I added a summary line.